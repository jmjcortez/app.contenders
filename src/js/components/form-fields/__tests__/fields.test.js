import React from 'react';
import { FieldInput } from '../index';
import { shallow } from 'enzyme';

describe('Field input test', () => {
    it('should render correctly with no props with input prop', () => {
        const input = {
            value: 'test'
        }
        const component = shallow(<FieldInput input={input} />);
        
        expect(component).toMatchSnapshot();
    });
});