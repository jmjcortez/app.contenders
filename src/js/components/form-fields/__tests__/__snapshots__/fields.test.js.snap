// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Field input test should render correctly with no props 1`] = `
<FormControl
  value="test"
/>
`;

exports[`Field input test should render correctly with no props with input prop 1`] = `
<FormControl
  value="test"
/>
`;
